File: examples-sdk/java/507-villog-bench-file-drain/Main.java
// 507-villog-bench-file-drain — Java SDK equivalent
// Compile: vil compile --from java --input 507-villog-bench-file-drain/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}

File: examples-sdk/csharp/507-villog-bench-file-drain/Main.cs
// 507-villog-bench-file-drain — C# SDK equivalent
// Compile: vil compile --from csharp --input 507-villog-bench-file-drain/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();
